.github/workflows/dispatch.yml

```diff
@@ -12,43 +12,53 @@ env:
   # With ~180 repos, this means we'll run around 9 instances of the copy workflow in parallel.
   # This is (hopefully) sufficient to prevent us from triggering GitHub Action's abuse detection mechanism.
   MAX_REPOS_PER_WORKFLOW: 20
-  FILES: '[ ".github/workflows/automerge.yml", ".github/workflows/go-test.yml", ".github/workflows/go-check.yml", ".github/workflows/releaser.yml", ".github/workflows/release-check.yml", ".github/workflows/tagpush.yml" ]' # a JSON array of the files to distribute
 
 jobs:
   matrix:
     name: Trigger copy workflows
     runs-on: ubuntu-latest
     outputs:
-      targets: ${{ steps.set-matrix.outputs.targets }}
+      batches: ${{ steps.set-matrix.outputs.batches }}
     steps:
       - uses: actions/checkout@v2
       - id: set-matrix
         run: |
-          CONFIG=config.json
-          if [[ $GITHUB_REF == refs/heads/testing ]]; then
-            CONFIG=config-testing.json
-          fi
-          TARGETS=$(jq '. | _nwise(${{ env.MAX_REPOS_PER_WORKFLOW }})' $CONFIG | jq -sc '. | to_entries')
-          echo "::set-output name=targets::$TARGETS"
+          TARGETS=()
```

Comment:
Why is this capitalized? Is it an env? Otherwise, it would probably be nicer to not capitalize this var.